content/en/providers/getting-started/integration_checklist/_index.md

```diff
@@ -35,7 +35,7 @@ Resource Centre integrated into the EGI Federation.
 
 1. Security:
 
-   - HTC: [pakiti](https://wiki.egi.eu/wiki/EGI_CSIRT:Pakiti_client) is
+   - HTC: [pakiti](../../../internal/security-coordination/monitoring/pakiti) is
      installed and the outcome of the EGI CSIRT assessment is positive;
    - Cloud: the
      [EGI security Survey](https://www.surveymonkey.com/r/Cloud_Security_Questionnaire_for_Resource_Centres)
```

Comment:
Just noticed this link is also gone. I don't recall the latest URL

Reply:
I've been already asking Ale about the new survey url, we will see what we can find.

Reply:
OK, so in fact the survey got migrated last year, this reference just got forgotten.
The new link was documented in https://go.egi.eu/sec05